engine/common/rpc/convert/events.go

```diff
@@ -258,3 +261,26 @@ func BlockEventsToMessage(block flow.BlockEvents) (*accessproto.EventsResponse_R
 		Events:         eventMessages,
 	}, nil
 }
+
+// GetConversionEventEncodingVersion returns the appropriate event encoding version for conversion.
+// Execution node events payloads are always encoded in CCF. If eventEncodingVersionValue is JSON-CDC,
+// CFF-encoded payloads should be converted to JSON-CDC, and CFF is returned as the format to convert from.
+// If eventEncodingVersionValue is CFF, payloads will not be converted and will be left as is. In this case,
+// JSON-CDC is returned as the format to convert from, which will be ignored in converters, and payloads
+// will be returned as-is.
+func GetConversionEventEncodingVersion(eventEncodingVersionValue *entities.EventEncodingVersionValue) entities.EventEncodingVersion {
```

Comment:
this is a little confusing. What do you think about switching to use `entities.EventEncodingVersion` everywhere and explicitly specify the default encoding.